common/src/test/java/org/keycloak/common/ProfileTest.java

```diff
@@ -114,6 +114,18 @@ public void checkErrorOnBadConfig() {
         }
     }
 
+    @Test
+    public void wrongProfileInProperties() {
+        Properties properties = new Properties();
+        properties.setProperty("keycloak.profile", "experimental");
+
+        try {
+            Profile.configure(new PropertiesProfileConfigResolver(properties));
+        } catch (ProfileException e) {
+            Assert.assertEquals("Invalid profile 'experimental' specified via 'keycloak.profile' property", e.getMessage());
+        }
```

Comment:
There's a missing assertion that the configure method will fail. Rather than adding that, this could be done as an assertThrows:

```suggestion
        Assert.assertEquals("Invalid profile 'experimental' specified via 'keycloak.profile' property",
                assertThrows(ProfileException.class,
                        () -> Profile.configure(new PropertiesProfileConfigResolver(properties))).getMessage());
```

Reply:
Yep, better. Thanks